draft-ietf-tls-tls13.md

```diff
@@ -3974,6 +3974,13 @@ even with the same secret. In a 0-RTT exchange, Derive-Secret is
 called with four distinct transcripts; in a 1-RTT only exchange
 with three distinct transcripts.
 
+The complete transcript passed to Derive-Secret is always taken from the
+following sequence of handshake messages, starting at the first ClientHello and
+including only those messages that were sent: ClientHello, HelloRetryRequest,
+ClientHello, ServerHello, EncryptedExtensions, Server CertificateRequest, Server
+Certificate, Server CertificateVerify, Server Finished, EndOfEarlyData, Client
```

Comment:
And there is no problem with the client sending an EndOfEarlyData that the server can't decrypt [to include in its transcript] because the client MUST NOT send one if the server rejects early data.  But do we need to mention that explicitly here?